students/alinafe/session02/print_grid.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+"""
+Grid Printer Exercise:
+"""
+
+def print_grid():
+
```

Comment:
this is fine, but does violate the DRY principle (don't repeat yourself).
soring the horizontal and vertical lines in a variable would be a way to avoid that.